lib/utils/timetable.ts

```diff
@@ -213,3 +213,18 @@ export const camelCaseToKebabCase = (str: string): string => {
 export const kebabCaseToCamelCase = (str: string): string => {
   return str.replace(/-./g, (x) => x[1].toUpperCase());
 };
+
+const dateIsInThePast = (date: string) => new Date(date).getTime() < new Date().getTime()
+
+const dateIsInTheFuture = (date: string) => new Date(date).getTime() > new Date().getTime()
+
+export const isValidEvent = (event: DevelopmentPlanTimetable): boolean => {
+
+  const endDateIsValid = !event.endDate || dateIsInTheFuture(event.endDate)
+
+  return (
+    dateIsInThePast(event.startDate) &&
+    endDateIsValid &&
+    event.developmentPlanEvent !== TimetableEventKey.TimetableUpdated
```

Comment:
I am assuming we still want to exclude the timetable-updated events

Reply:
changed this util function so that it can handle both devPlans and devPlanEvents